src/main/java/uk/gov/justice/hmpps/prison/api/resource/AgencyResource.java

```diff
@@ -278,13 +280,20 @@ public List<Location> getAgencyEventLocations(
         @ApiResponse(responseCode = "400", description = "Invalid request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
         @ApiResponse(responseCode = "404", description = "Requested resource not found.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
         @ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})})
-    @Operation(summary = "List of locations for agency where events (appointments, visits, activities) are being held.", description = "List of locations for agency where events (appointments, visits, activities) are being held.")
+    @Operation(summary = "List of location summaries for agency where events (appointments, visits, activities) are being held.",
+        description = "List of location summaries for agency where events (appointments, visits, activities) are being held.")
     @ReferenceData(description = "Agency data is considered non-sensitive")
     @GetMapping("/{agencyId}/eventLocationsBooked")
     @SlowReportQuery
-    public List<Location> getAgencyEventLocationsBooked(
-        @PathVariable("agencyId") @Parameter(required = true) final String agencyId, @RequestParam("bookedOnDay") @org.springframework.format.annotation.DateTimeFormat(iso = DateTimeFormat.ISO.DATE) @Parameter(description = "Filter list to only return locations which prisoners will be attending on this day", required = true) final LocalDate date,
-        @RequestParam(value = "timeSlot", required = false) @Parameter(description = "Only return locations which prisoners will be attending in this time slot (AM, PM or ED, and bookedOnDay must be specified)", schema = @Schema(implementation = String.class, allowableValues = {"AM", "PM", "ED"})) final TimeSlot timeSlot
+    public List<LocationSummary> getAgencyEventLocationsBooked(
```

Comment:
Is this the breaking change you mentioned in standup? (I was thinking you could have created a new endpoint instead of changing the existing one then you could move the clients over at leisure)

Reply:
I got a bit confused - there are 2 endpoints that are very similarly named (doh!)